test/testcafe/spec/EnrollFactorPassword_spec.js

```diff
@@ -31,16 +31,16 @@ test(`should have both password and confirmPassword fields and both are required
   // fields are required
   await enrollPasswordPage.clickNextButton();
   await enrollPasswordPage.waitForErrorBox();
-  await t.expect(enrollPasswordPage.getPasswordError()).eql('This field cannot be left blank');
-  await t.expect(enrollPasswordPage.getConfirmPasswordError()).eql('This field cannot be left blank');
+  await t.expect(enrollPasswordPage.getPasswordError()).contains('This field cannot be left blank');
```

Comment:
hmm, why cannot use `eql`?

Reply:
The precompiled templates sometimes have an extra whitespace in them (if the template in the sourcefile has line breaks and empty spaces). Most of these are stripped, but there are some cases where an extra space will stay in. (Input label is one of these cases)

Reply:
`getPasswordError` tries to get inner text of following `p` element.
Wouldn't expect extra line break ? 
so maybe `trim` whilespace so that we can still use `eql` ?

```
<p id="input-container-error6" class="okta-form-input-error o-form-input-error o-form-explain" role="alert"><span class="icon icon-16 error-16-small" role="img" aria-label="Error"></span>This field cannot be left blank</p>
```

Reply:
I'll take another look at this

Reply:
can be separate PR if it's not quick fix.

Reply:
I investigated, this is already fixed/tightened up on the courage side, so this must have been from an earlier iteration. I'll put it back to `eql`